src/api/spec/models/review_spec.rb

```diff
@@ -39,6 +68,72 @@
     it { is_expected.to match_array([review_unassigned1, review_unassigned2]) }
   end
 
+  describe '.set_associations' do
+    context 'with valid attributes' do
+      it 'sets user association when by_user object exists' do
+        review = Review.create(by_user: user.login)
+        expect(review.valid?).to eq(true)
```

Comment:
I dont think that a factory would be useful here. This is about testing that when you call `Review.create` it populates the entity columns so IMO its better to call `Review.create` explicitely.